wled00/wled.h

```diff
@@ -354,7 +354,7 @@ WLED_GLOBAL bool notifyButton _INIT(false);                       // send if upd
 WLED_GLOBAL bool notifyAlexa  _INIT(false);                       // send notification if updated via Alexa
 WLED_GLOBAL bool notifyMacro  _INIT(false);                       // send notification for macro
 WLED_GLOBAL bool notifyHue    _INIT(true);                        // send notification if Hue light changes
-WLED_GLOBAL bool notifyTwice  _INIT(false);                       // notifications use UDP: enable if devices don't sync reliably
+WLED_GLOBAL uint8_t ubUdpNumRetries _INIT(1);                     // Number of UDP messages to send. Increase to increase reliability
```

Comment:
Why did you choose default of 1 retry? Current behaviour is without any retries.

Reply:
This was an accidental carry over from a previous thought process. Good catch this should be zero.